src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -158,36 +149,65 @@ public class GT_MetaTileEntity_ExtremeIndustrialGreenhouse
      * Changing T one number down will buff the EIG twice, as well as changing it up will nerf the EIG twice
      * (That is because accelerators are imperfectly scaled in game LV = 2x, MV = 4x, ...)
      */
-    public static final int EIG_BALANCE_IC2_ACCELERATOR_TIER = 5; // IV
+    public static final int EIG_BALANCE_IC2_ACCELERATOR_TIER = GTVoltageIndex.IV;
+    public static final int EIG_BALANCE_REGULAR_MODE_MIN_TIER = GTVoltageIndex.EV;
+    public static final int EIG_BALANCE_IC2_MODE_MIN_TIER = EIG_BALANCE_IC2_ACCELERATOR_TIER + 1;
+    public static final double EIG_BALANCE_MAX_FERTILIZER_BOOST = 4.0d;
+    public static final int EIG_BALANCE_WEED_EX_USAGE_BEGINS_AT = 1000;
+    public static final int EIG_BALANCE_WATER_USAGE_PER_SEED = 1000;
 
-    /***
-     * All crops above this threshold will die if WEED-EX 9000 isn't supplied.
-     * Consider changing this value after making changes to the EIG_BALANCE_IC2_ACCELERATOR_TIER.
-     */
-    private static final int EIG_CROP_LIMIT_FOR_WEEDEX9000_REQUIREMENT = 1000;
+    private static final Fluid WEEDEX_FLUID = Materials.WeedEX9000.mFluid;
+    private static final Item FORESTRY_FERTILIZER_ITEM = GameRegistry.findItem("Forestry", "fertilizerCompound");
```

Comment:
Attempting to search mod item on class load may fail, cause there's no guarantees, that mod is loaded at that point.

Reply:
This is how it uses to work, I just grouped it with the rest of the balance related constants.
https://github.com/GTNewHorizons/KubaTech/blob/a17f0c238deadc7b107b37a5a852387a267cbb1f/src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java#L189

I've moved fertilizer item declarations to the post-init recipe loader anyway so it should be fine now.